renku/service/views/apispec.py

```diff
@@ -39,7 +39,7 @@
 gitlab_token_scheme = {"type": "apiKey", "name": "Authorization", "in": "header"}
 
 TOP_LEVEL_DESCRIPTION = """
-This is the API specification of the renku core service. The API follows the
+This is the API specification of the renku core service. The API loosely follows the
```

Comment:
I honestly don't think we follow JSON-RPC at all. I wonder if we should just remove that.

Reply:
Right, I guess this only applies to the response format. That still *loosely* follows JSON-RPC 2.0.
I'll remove this sentence and rephrase a bit the content below. Let me know if you think it's better, or if this section should be overhauled.